shared/src/main/scala/com/cibo/evilplot/plot/Plot.scala

```diff
@@ -66,6 +66,8 @@ final case class Plot(
   private[plot] def +:(component: FacetedPlotComponent): Plot =
     copy(components = component +: components)
 
+  def appendComponent(component: FacetedPlotComponent): Plot = this :+ component
```

Comment:
keeping both the symbolic ones and the named ones would probably be annoying to us later on. I'm finding it kind of difficult to think of adjectives to differentiate them other than `interior` or `exterior` but I'm not sure those are particularly intuitive. I guess anything that conveys the positioning aspect of this well would be ideal (I'm thinking "append" and "prepend" don't convey this properly either)

Reply:
Aye, I'm struggling with too since components act differently depending on whether they are overlay/background or side-based. I figured sticking close to the implementation (a list of components) at least conveyed that the components would render "before" or "after" others (regardless of whether that becomes "above" & "below"; or "interior" & "exterior")